src/main/java/ch/hearc/cafheg/business/allocations/AllocationService.java

```diff
@@ -171,16 +183,18 @@ public Allocataire updateAllocataire(long allocataireId, String newNom, String n
 
         // Update should be done only if a changed is detected
         if (!nameChanged && !firstnameChanged) {
+            logger.info("Aucun changement détecté pour l'allocataire ID: {}", allocataireId);
```

Comment:
This is not an intended behavior, it should be pointed as a warning
```suggestion
            logger.warn("Aucun changement détecté pour l'allocataire ID: {}", allocataireId);
```